### PR TITLE
Correct the return value of BIO_get_ktls_*().

### PR DESCRIPTION
**Pull request by bsdjhb**
BIO_get_ktls_send() and BIO_get_ktls_recv() are documented as
returning either 0 or 1.  However, they were actually returning the
internal value of the associated BIO flag for the true case instead of
1.

### CODE REVIEW

File: crypto/bio/bss_conn.c [hunk 1]
```diff
@@ -546,9 +546,9 @@ static long conn_ctrl(BIO *b, int cmd, long num, void *ptr)
             BIO_set_ktls_flag(b, num);
         break;
     case BIO_CTRL_GET_KTLS_SEND:
-        return BIO_should_ktls_flag(b, 1);
+        return BIO_should_ktls_flag(b, 1) ? 1 : 0;
```
Comment (t8m) [inline crypto/bio/bss_conn.c:549]: We usually use `!= 0` in such case instead of the ternary `?` operator.

Comment (bsdjhb) [reply crypto/bio/bss_conn.c:549]: Ok, I was copying from BIO_CTRL_EOF a few lines above which does the ternary.  I'm happy to change it to != 0 instead though as that is what I normally do myself.

Comment (t8m) [reply crypto/bio/bss_conn.c:549]: You can fix that place as well to make it uniform, if you want.

**Review by slontis — APPROVED** [review]

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
Merged to master. Thank you for the contribution.

**Pull request closed without merging**